src/Symfony/Component/Validator/Tests/Constraints/FileTest.php

```diff
@@ -22,11 +22,11 @@ class FileTest extends TestCase
     /**
      * @dataProvider provideValidSizes
      */
-    public function testMaxSize($maxSize, $bytes, $binaryFormat)
+    public function testMaxSize($maxSize, $bytes, $binaryFormat, $exceeds32BitMax)
     {
         $file = new File(['maxSize' => $maxSize]);
 
-        $this->assertSame($bytes, $file->maxSize);
+        $this->assertSame($exceeds32BitMax && 4 === \PHP_INT_MAX ? (string) $bytes : $bytes, $file->maxSize);
```

Comment:
This uses the wrong constant. The max integer is not 4...

Reply:
oops, indeed